invenio_openaire/loaders.py

```diff
@@ -194,9 +195,9 @@ def grantxml2json(self, grant_xml):
         internal_id = "{0}::{1}".format(funder['doi'], code)
         eurepo_id = \
             "info:eu-repo/grantAgreement/{funder}/{program}/{code}/".format(
```

Comment:
This should be prefixed with `u"..."` to work with unicode strings. [Similar literals](https://github.com/inveniosoftware/invenio-openaire/pull/48/files#diff-9c5635ece921a333e2d97b6e925b7ff1R195) as well

Reply:
I think this already achieves the same: https://github.com/inveniosoftware/invenio-openaire/pull/48/files#diff-9c5635ece921a333e2d97b6e925b7ff1R39